README.md

```diff
@@ -15,21 +15,41 @@ Examples for [Banuba SDK on Android](https://docs.banuba.com/face-ar-sdk-v1/andr
 2. Get the latest Banuba SDK archive, [Banuba trial client token](https://www.banuba.com/facear-sdk/face-filters).
 To receive full commercial license from Banuba - please fill in our form on [form on banuba.com](https://www.banuba.com/face-filters-sdk) website, or contact us via [info@banuba.com](mailto:info@banuba.com).
 3. Copy `.aar` file from the Banuba SDK archive into `libs` dir:
-   >`banuba_effect_player-release.aar` => `libs/`
-4. Copy `include/bnb` directory into `libs/bnb_sdk/`:
-   >`include/bnb` => `libs/bnb_sdk/bnb`
-5. Download and copy folders of the needed effects into `app/src/main/assets/effects` directory. The effects can be downloaded from [here](https://docs.banuba.com/face-ar-sdk-v1/overview/demo_face_filters).
-6. Get the Agora Video SDK archive for android (v4.0.0-beta.2) from [here](https://docs.agora.io/en/extension_customer/downloads?platform=Android).
-7. Copy `jar` files from the Agora Video SDK archive into `libs` dir:
-   >`agora-rtc-sdk.jar` => `libs/`
-8. Copy architecture folders from the Agora Video SDK archive into `jniLibs` dir:
-   >`arm64-v8a`, `armeabi-v7a` and `x86_64` => `libs/jniLibs/`
-9. Copy Agora SDK headers (`*.h` files) into `libs/agora_rtc_sdk/include`:
-   >`low_level_api/include` => `libs/agora_rtc_sdk/include`
-10. Copy and Paste your Banuba client token into appropriate section of `app/src/main/java/com/banuba/sdk/agorapluginexample/ClientToken.kt` with " " symbols. For example: BANUBA_CLIENT_TOKEN = "place_your_token_here"
-11. Visit [agora.io](https://console.agora.io) to sign up and get token, app ID and channel ID.
-12. Copy and Paste your Agora token, app ID and channel ID into appropriate section of `app/src/main/java/com/banuba/sdk/agorapluginexample/ClientToken.kt` with " " symbols. For example: AGORA_APP_ID = "place_your_token_here"
-13. Open the project in Android Studio and run the necessary target using the usual steps.
+   >`banuba_effect_player-release.aar` => [`libs/`](./libs)
+4. Copy `include/bnb` directory into [`libs/bnb_sdk/`](./libs/bnb_sdk):
+   >`include/bnb` => [`libs/bnb_sdk/`](./libs/bnb_sdk)
+5. Download the effects you want from [HERE](https://docs.banuba.com/face-ar-sdk-v1/overview/demo_face_filters). Unzip the downloaded `.zip` files with effects to a folder [`app/src/main/assets/effects`](./app/src/main/assets/effects). Each effect is one folder.
```

Comment:
Download the effects you need from ....
"Each effect is one folder." ->
Each unpacked effect should be put into a separate folder. The folder name will be the effect name for loading.